static/app/views/acceptOrganizationInvite.spec.jsx

```diff
@@ -25,28 +28,25 @@ describe('AcceptOrganizationInvite', function () {
       existingMember: false,
     });
 
-    const wrapper = mountWithTheme(
-      <AcceptOrganizationInvite params={{memberId: '1', token: 'abc'}} />
-    );
-
-    const joinButton = wrapper.find('Button[aria-label="join-organization"]');
+    render(<AcceptOrganizationInvite params={{memberId: '1', token: 'abc'}} />);
 
-    expect(joinButton.exists()).toBe(true);
-    expect(joinButton.text()).toBe('Join the test-org organization');
+    const joinButton = getJoinButton();
+    expect(joinButton).toBeInTheDocument();
 
     const acceptMock = MockApiClient.addMockResponse({
       url: '/accept-invite/1/abc/',
       method: 'POST',
     });
 
-    joinButton.simulate('click');
+    userEvent.click(joinButton);
```

Comment:
```suggestion
    userEvent.click(screen.getByRole('button', {name: 'Join the test-org organization'}));
```

if the button is not found, it will throw here, so we don't need to prior check if the element is in the document 😉